<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdSimulator</class>
 <widget class="QWidget" name="mdSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mdSimulator</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Lennard-Jones Fluid Simulation</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="simulationView">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>590</width>
     <height>470</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>91</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>N X N atoms, insert N</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nAtomInp"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_nAtom">
        <property name="text">
         <string>Density</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="densityInp"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_dnsity">
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="temperatureInp"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_TimeStep">
        <property name="text">
         <string>Time Step</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="timeStepInp">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="defaultButton">
        <property name="text">
         <string>Set Default</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="setParamButton">
        <property name="text">
         <string>Set Parameters</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="helpButton">
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
      <widget class="QPushButton" name="simulateButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tab_plot">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>480</y>
     <width>591</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QCustomPlot" name="totalEnergyPlot">
    <attribute name="title">
     <string>Total Energy Plot</string>
    </attribute>
   </widget>
   <widget class="QCustomPlot" name="KEPlot">
    <attribute name="title">
     <string>Kinetic Energy Plot</string>
    </attribute>
   </widget>
   <widget class="QCustomPlot" name="PEPlot">
    <attribute name="title">
     <string>Potential Energy Plot</string>
    </attribute>
   </widget>
  </widget>
  <zorder>simulationView</zorder>
  <zorder>label</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>tab_plot</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
